marketplace_docs/templates/Fabric/files/etc/update-motd.d/99-one-click

```diff
@@ -2,14 +2,12 @@
 #
 # Configured as part of the DigitalOcean 1-Click Image build process
 
-myip=$(hostname -I | awk '{print$1}')
 cat <<EOF
 ********************************************************************************
 
-This is the text that will be shown in the MOTD users see when they log in via
-SSH.  This copy should include links to documentation or product informaiton
-along with any information that would helpful to users.
+If you have any queries visit https://chat.zulip.org to get help from the Zulip community.
+For enterprise plans and support mail at support@zulipchat.com.
```

Comment:
I think linking to https://zulipchat.com/plans is probably better than the email address.

@rishig any other thoughts on this text, which is what will show up every time folks logic to systems created using this 1-click installer?  (So pretty highly visible)

Reply:
@timabbott Yup. They have recommended it like that. https://github.com/zulip/marketplace-partners/blob/master/marketplace_docs/build-an-image-fabric.md#running-commands-on-first-login